test/analysis/expected/test.csv

```diff
@@ -0,0 +1,8 @@
+,Unnamed: 0,test,test.1,test.2
```

Comment:
What is the column header telling us here?

Reply:
The weird thing about this, is that I am setting the edge-pairs to be the index each time I make a dataframe, so I don't think I can get rid of the unnamed column unless I do index=False when I save the file